src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,66 @@
+export default class PokemonClient {
+  constructor() {
+    this.API_BASE = "https://pokeapi.co/api/v2";
+  }
+
+  async fetchAllPokemons() {
+    try {
+      const pokemons = await fetch(
+        `${this.API_BASE}/pokemon?limit=100000&offset=0`
+      );
+      const pokemonsJson = await pokemons.json();
+
+      return pokemonsJson.results;
+    } catch {
+      (err) => console.log(err);
+    }
+  }
+
+  async fetchPokemonNameById(id) {
+    try {
+      const pokemonsList = await this.fetchAllPokemons();
+      if (pokemonsList[id].name) {
```

Comment:
the list we get has offset of 1 index, first pokemon (id 1) is in first index (0) so there's a bug here :)

Reply:
fixed it :)

Reply:
fixed with previous changes